apps/expo/src/app/workout-completion/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { Text, View } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { router, useLocalSearchParams } from 'expo-router'
+
+import { AntDesign, Ionicons } from '@expo/vector-icons'
+
+import RotatingBarbellIcon from '~/components/notif/RotatingBarbellIcon'
+import { TExercise } from '~/components/tracker/workout-tracker'
+import Button from '~/components/ui/button/button'
+import { useGlobalContext } from '~/context/global-context'
+import colors from '~/styles/colors'
+import { formatForWorkoutCompletion } from '~/utils/timerFormatter'
+import { api } from '~/utils/trpc/api'
+
+const WorkoutCompletion: React.FC = () => {
+  const { userData } = useGlobalContext()
+  const { data, isFetching } = api.user.getUserWorkoutHistory.useQuery({
+    userId: userData!.id,
+  })
+
+  const { workoutName, exercises, duration, dateFinished } = useLocalSearchParams() // TODO: type this
```

Comment:
Try to resolve the TODO if possible.

Reply:
Some TODOs are very time consuming but this one is rather quick so I will implement this